src/desktop/apps/authentication/__tests__/helpers.jest.ts

```diff
@@ -21,25 +31,83 @@ jest.mock("sharify", () => {
     },
   }
 })
+const sd = require("sharify").data
```

Comment:
Again not sure how much this actually matters but might want to mock this or restore values that are set in individual tests in an `after` hook

Reply:
I'm pretty sure these types of mocks are cleaned up after the file is run, haven't seen issues with leakage in the past. Attempted to cleanup, but it looks like sd resolves as an object after it is called, and doesn't identify as a mock anymore.

Reply:
👍 yeah i was thinking that jest is smart about this. I wanted to bring up because I remember Eloy mentioning something about it a while back.